test/unit/store/student-store.js

```diff
@@ -71,6 +73,35 @@ describe('StudentStore', () => {
         });
     });
 
+    it('should update student items on edit item date', () => {
+        let tomorrow = moment().tz(config.get('timezone'));
+        return addAction('NEW_CHECKOUT', {
+            studentId: student.id,
+            equipment: [
+                {
+                    address: items[0].address
+                },
+                {
+                    address: items[2].address
+                }
+            ]
+        }).then(() => {
+            assert.lengthOf(student.items, 2);
+            tomorrow.add(2, 'd');
+            return addAction('CHANGE_ITEM_DUEDATE', {
+                dueDate: tomorrow.format('YYYY-MM-DD'),
+                studentId: student.id,
+                itemAddress: items[0].address
+            });
+        }).then(() => {
+            tomorrow.hour(17).minute(0).second(0);
```

Comment:
NACK: change this to use `checkin.due_hour` and `checkin.due_minute` from config